fvm/evm/testutils/accounts.go

```diff
@@ -0,0 +1,135 @@
+package testutils
+
+import (
+	"bytes"
+	"crypto/ecdsa"
+	"io"
+	"math/big"
+	"sync"
+	"testing"
+
+	gethCommon "github.com/ethereum/go-ethereum/common"
+	gethTypes "github.com/ethereum/go-ethereum/core/types"
+	gethCrypto "github.com/ethereum/go-ethereum/crypto"
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/atree"
+
+	"github.com/onflow/flow-go/fvm/evm/emulator"
+	"github.com/onflow/flow-go/fvm/evm/emulator/database"
+	"github.com/onflow/flow-go/fvm/evm/types"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// address:  658bdf435d810c91414ec09147daa6db62406379
+const EOATestAccount1KeyHex = "9c647b8b7c4e7c3490668fb6c11473619db80c93704c70893d3813af4090c39c"
+
+type EOATestAccount struct {
+	address gethCommon.Address
+	key     *ecdsa.PrivateKey
+	nonce   uint64
+	signer  gethTypes.Signer
+	lock    sync.Mutex
+}
+
+func (a *EOATestAccount) Address() types.Address {
+	return types.Address(a.address)
+}
+
+func (a *EOATestAccount) PrepareSignAndEncodeTx(
+	t testing.TB,
+	to gethCommon.Address,
+	data []byte,
+	amount *big.Int,
+	gasLimit uint64,
+	gasFee *big.Int,
```

Comment:
I believe this should be named gasPrice

Reply:
nice find, updated at [c848b92](https://github.com/onflow/flow-go/pull/4927/commits/c848b9212ca242dd68b095c4a3cd63058af8c377)